### DISCUSSION TITLE
Kritischer Fehler Version 2.1

### DISCUSSION THREAD

**Discussion by madlin24** [affiliatetheme/affiliatetheme#26] [General]
Hallo. Wir haben das Affiliate Theme Version 2.0 auf mehreren Seiten im Einsatz. Aktuell wird gerade die Version 2.1 angeboten. Nach dem Einspielen kommt es bei den meisten Seiten zu einem kritischen Fehler. Hat jemand die gleichen Erfahrungen gemacht? Es wird auch nicht angezeigt, was das Update beinhaltet. Wir haben die alte Version wieder eingespielt und es läuft. Wir haben zur Sicherheit die automatische Aktualisierung deaktiviert. Hat jemand ein Tipp?

**Comment by endcoreCL** [comment]
Hey, um dir zu helfen benötige ich bitte die Fehlermeldung. Dann kann ich mir das gerne mal ansehen.

**Reply by davsaisa** [reply]
Moin Christian, nach dem Theme-Update kommt ein kritischer Fehler + white Screen. Ich dachte erst es wären Plugins - ACF löst den Fehler aus allerdins erst mit dem Theme-Update.

**Comment by madlin24** [comment]
Hallo. Danke für die schnelle Antwort. Nach einigen Versuchen konnte ich folgende Fehlermeldung dem System entlocken:

Fatal error: Allowed memory size of 268435456 bytes exhausted (tried to allocate 8192 bytes) in /www/htdocs/xxx/class-wp-query.php on line 627

Zudem hatte ich noch eine weitere Meldung bei der Version 2.0 (nachdem ich die alte Version zurückgespielt habe):

Die Funktion acf_get_value wurde fehlerhaft aufgerufen. Advanced Custom Fields PRO â€“ Es wurde mindestens ein Versuch festgestellt, ACF-Feldwerte abzurufen, bevor ACF initialisiert wurde. Dies wird nicht unterstÃ¼tzt und kann zu fehlerhaften oder fehlenden Daten fÃ¼hren.

Kannst Du damit etwas anfangen?
Danke im Voraus!

**Comment by Chauffe** [comment]
Ja, bei mir kam auch ein kritischer Fehler nach dem Update. Bin mir nicht ganz sicher, aber der Fehler müsste auch im Zusammenhang mit class-wp-query.php aufgetreten sein.

**Comment by madlin24** [comment]
Gibt es denn eine Lösung für das Update? Was war denn in dem Update 2.1 enthalten? Wird denn überhaupt an dem Theme gearbeitet?
